client/src/components/SignupForm/SignupForm.tsx

```diff
@@ -53,7 +55,11 @@ export function SignupForm({ onSubmit }: Props) {
   };
 
   return (
-    <ColumnLayout className="bg-napari-light p-6 screen-495:p-12">
+    <ColumnLayout
+      className="bg-napari-light p-6 screen-495:p-12"
+      // Use 3-column layout instead of 4-column layout.
+      classes={isHome ? { fourColumn: 'screen-1150:grid-cols-napari-3' } : {}}
+    >
```

Comment:
in terms of the actual API though I think the following appeals to me more:
```suggestion
<ColumnLayout className={clsx('bg-napari-light p-6 screen-495:p12', isHome && '~screen-1150:grid-cols-napari-4 screen-1150:grid-cols-napari-3')} />
```

basically `~` before a class name signifies that it should be removed

Reply:
I can see this working, although if we're going to use a special operators, we might as well use the important operator `!`:

```tsx
<ColumnLayout className={clsx(
  'bg-napari-light p-6 screen-495:p12',
  isHome && 'screen-1150:!grid-cols-napari-3'
)} />
```

But using `!important` is generally not recommended. Either way, this is good feedback and is something we can possibly look into during the quality sprint.